### ISSUE TITLE
Docs on new super-merge stuff

### ISSUE THREAD

**Issue by andrew-davie** [dasm-assembler/dasm#81]
Do I need to go through all the recently committed code to find/understand the changes, or is there some documentation to which I can refer and be sure I caught them all?  Maybe we could have some way of standardising required additions to the documentation, perhaps by assigning myself to an issue with the documentation in it?  Just trying to figure out how to handle this so it's done routinely.

**Comment by dionoid** [comment]
Good point. IMO the super-merge was a one-time thing to get the changes from @thomas374b into the master. From now on, we should only allow pull requests that fix one issue, or add one feature. This is a best practice which makes it easier to review, test and verify. If a future PR adds a new command line switch (e.g. that "-m" switch that was added to maximize the generated file), then I think the same PR must also include changes to the manual/user documentation.

Back to your question: Closing the issues which were fixed by the super-merge is still on my list. So far, it looks like the super-merge fixed a lot of bugs (#53, #32, #55, #41, #62, #61, #56, #63, #64, #65, #66, #60, #68 and #69) and added support for mc68hc908. The "-m" switch was added, which I think so far is the only new switch.

Btw: Once the User Manual is finished, I think other people should also be able to make additions when e.g. switches are added/changed. Question: what is the preferred tool to create PDF from the LaTeX .tex files? Are the online tools any good?

**Comment by andrew-davie** [comment]
There was also the -R switch. I have documented both in db3a403f44a6a7dc0866eefe606010069bf99414

Others are welcome to make changes -- hopefully we can keep a consistent look/feel/style going here though.
I'm only doing it all now because it seemed much more efficient to tackle it in one go, and then defer incremental updates to the rest of the team (or me). I don't "own" the manual; it's a shared file just like all the others.

TexStudio is my go-to for latex editing and PDF production.
Basically I edit the latex source files, push one button and it's done.
I'm pretty sure it could be automated - yes, but that's not something I have experience with.

**Comment by dionoid** [comment]
From the super-merge (which mainly contains bug fixes), I found the -R and -m switches to be added, which you already found.
Other than the fact that mc68hc908 is supported, I found a change to support 'swapping the endian' for wide-char strings, which might be something to add to the user manual:

`DC.w "unicode" ; bytes order same as with current 'processor'`
`DC.s "unicode" ; opposite byte order ; 's' ~ abbrev. for 'short swap'`

See issue https://github.com/dasm-assembler/dasm/issues/65

**Comment by dionoid** [comment]
@andrew-davie Are you okay with closing this issue?

**Comment by andrew-davie** [comment]
Yep, I think everything is documented.

**Comment by dionoid** [comment]
Closing this issue
